libr/asm/arch/gb/gbasm.c

```diff
@@ -33,7 +33,7 @@ static bool gb_parse_cb1 (ut8 *buf, const int minlen, char *buf_asm, ut8 base) {
 		buf[0] |= (ut8)i;
 		return true;
 	}
-	if (!strncmp (buf_asm, "[hl]", 4)) {
+	if (!strncmp (&buf_asm[minlen - 1], "[hl]", 4)) {
```

Comment:
use buf_asm + minlen - 1. Can minlen be 0? if so we have an underflow

Reply:
nope, it cannot "" is not valid instruction, also wouldn't pass through the switch-case from the caller